scripts/check_cie_button_exists/check_cie_button_exists_ios.py

```diff
@@ -0,0 +1,99 @@
+import requests
+import re
+import os
+import sys
+
+cieHeaders = {
+    'User-Agent': 'Mozilla/5.0 (iPhone; CPU iPhone OS 11_0 like Mac OS X) AppleWebKit/604.1.38 (KHTML, like Gecko) Version/11.0 Mobile/15A372 Safari/604.1',
+    'Accept-Encoding': 'identity, deflate, compress, gzip, br',
+    'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8',
+    'Accept-Language': 'it-IT,it;q=0.8,en-US;q=0.5,en;q=0.3',
+    'Connection': 'keep-alive',
+    'Upgrade-Insecure-Requests': '1'
+}
+
+# As requests doesn't support JS, redirects stop, so a POST with the following payload is required
+ciePayload = {
+    "shib_idp_ls_exception.shib_idp_session_ss": "",
+    "shib_idp_ls_success.shib_idp_session_ss": "false",
+    "shib_idp_ls_value.shib_idp_session_ss": "",
+    "shib_idp_ls_exception.shib_idp_persistent_ss":	"",
+    "shib_idp_ls_success.shib_idp_persistent_ss":	"false",
+    "shib_idp_ls_value.shib_idp_persistent_ss": "",
+    "shib_idp_ls_supported": "",
+    "_eventId_proceed": ""
+}
+
+
+def requestCieAuthPage(
+        uri,
+        headers,
+        payload,
+        maxAttempts):
+
+    statusFlag = False
+    nAttempts = 0
+    response = None
+
+    # Do the required attempts
+    while not (nAttempts >= maxAttempts or statusFlag):
+        # do the requests
+        response = requests.get(
```

Comment:
If `get` or `post` request goes on timeout an exception will be raised. This is handled as a **ko** case.
Could be an improvement if we handle the timeout within the `nAttempts` already used for `status code !== 200`? Maybe with a sleep time between requests.

What do you think?

Reply:
Wouldn't it be better if we just add a longer timeout to requests?  it's just an input with a default value. We could set 1sec or whatever

Reply:
@GiovanniMancini I don't know. Pick a value you consider right. I guess wait 5/10 second could be fine.